### PR TITLE
[STARTAI-35] API | Write api for alcohol consumtion

### PR DESCRIPTION
**Pull request by JJerome-NM**
image

image

image

Response exemple

### {
    "schema": {
        "fields": [
            {
                "name": "index",
                "type": "integer"
            },
            {
                "name": 0,
                "type": "integer"
            }
        ],
        "primaryKey": [
            "index"
        ],
        "pandas_version": "1.4.0"
    },
    "data": [
        {
            "index": 0,
            "0": 11
        },
        {
            "index": 1,
            "0": 11
        },
        {
            "index": 2,
            "0": 10
        }
    ]
### }

### CODE REVIEW

File: app/api/alcohol_consumption_api/schema.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                {"name": "reason", "displayName": "Reason to choose this school", "type": "multi_select",
+                 "options": ["home", "reputation", "course", "other"]},
+                {"name": "failures", "displayName": "Number of past class failures", "type": "bool"},
+                {"name": "famsup", "displayName": "Family educational support", "type": "bool"},
+                {"name": "paid", "displayName": "Extra paid classes within the course subject", "type": "bool"},
+                {"name": "activities", "displayName": "Extra-curricular activities", "type": "bool"},
+                {"name": "higher", "displayName": "Wants to take higher education", "type": "bool"},
+                {"name": "internet", "displayName": "Internet access at home", "type": "bool"},
+                {"name": "famrel", "displayName": "Quality of family relationships", "type": "int"},
+                {"name": "freetime", "displayName": "Free time after school", "type": "int"},
```
Comment (ArsenOOO7) [inline app/api/alcohol_consumption_api/schema.py:37]: specify time unit here

File: app/api/alcohol_consumption_api/resource.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+from .alcohol_consumption_data_processing import AlcoholConsumptionDataProcessing
+from .schema import AlcoholConsumptionSchema
+
+from . import al_co_bp as app
+
+ALLOWED_EXTENSIONS = {'csv', 'json'}
+
+
+@app.get('/model_details')
```
Comment (ArsenOOO7) [inline app/api/alcohol_consumption_api/resource.py:14]: /fields

File: app/api/alcohol_consumption_api/schema.py (hunk 1 above)
Comment (JJerome-NM) [reply app/api/alcohol_consumption_api/schema.py:37]: image
But there's a number here, or am I not understanding something?

Comment (ArsenOOO7) [reply app/api/alcohol_consumption_api/schema.py:37]: ok, make it string and dropdown with these options
in request convert to int

**Pull request merged**